packages/cqrs/lib/src/cqrs.dart

```diff
@@ -166,4 +310,38 @@ class Cqrs {
       },
     ).timeout(_timeout);
   }
+
+  void _log(
+    CqrsMethod method,
+    _ResultType result, [
+    Object? error,
+    StackTrace? stackTrace,
+    List<ValidationError> validationErrors = const [],
+  ]) {
+    final log = switch (result) {
+      _ResultType.success => _logger?.info,
+      _ResultType.validationError => _logger?.warning,
+      _ => _logger?.severe,
```

Comment:
Personally I would write out all variants to be more explicit and get feedback when a new _ResultType is added